pkgs/applications/version-management/yadm/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl, git, bash }:
+
+let version = "1.04"; in
+let link = "https://raw.githubusercontent.com/TheLocehiliosan/yadm/${version}"; in
+stdenv.mkDerivation {
+  name = "yadm-${version}";
+  isLibrary = false;
```

Comment:
What is `isLibrary` and `isExecutable`? I've only seen those used with the cabal build system.